charts/rancher-aks-operator-crd/100.0.7+up1.0.8/templates/crds.yaml

```diff
@@ -122,6 +122,9 @@ spec:
                     vmSize:
                       nullable: true
                       type: string
+                    vnetSubnetID:
+                      nullable: true
+                      type: string
```

Comment:
The `vnetSubnetID` block seems to be absent from the original RC'd chart: https://github.com/rancher/charts/blob/48ee4d031658e4d2963ad9e2dc7787dcd63aba30/charts/rancher-aks-operator-crd/100.0.7%2Bup1.0.8-rc2/templates/crds.yaml#L119-L130

Is this a change we want in the released chart?

Reply:
I believe so. If you compare that rc with the latest release/v1.0 https://github.com/rancher/aks-operator/compare/v1.1.0-rc2...release/v1.0, you'll see the highlander team put in this PR https://github.com/rancher/aks-operator/pull/80 which adds the `vmSubnetId` to the node pool in the aks operator. The PR description specifies the fixes are needed in rancher 2.6

Reply:
Makes sense! Thanks for clarifying!